tests/visual/scheduler-angular-edit-dialog.html

```diff
@@ -0,0 +1,233 @@
+<!doctype html>
+<html lang="en" class="k-typography">
+<head>
+    <title>Scheduler Angular Edit Dialog</title>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" href="../../packages/default/dist/all.css" />
+    <link rel="stylesheet" href="assets/styles.css" />
+    <script src="assets/scripts.js"></script>
+
+    <style>
```

Comment:
The test's screenshots are not looking okay. This may be caused of the positioning styles of the dialog and should be handled in the test's styles, just for the test purpose. Please, check this https://github.com/telerik/kendo-themes/blob/develop/tests/visual/dialog.html#L16